source/gui/startupDialogs.py

```diff
@@ -221,28 +217,25 @@ def __init__(self, parent):
 		mainSizer.Fit(self)
 		self.CentreOnScreen()
 
-	def _createLicenseAgreementGroup(self) -> wx.StaticBoxSizer:
+	def _createLicenseAgreementGroup(self) -> wx.BoxSizer:
+		sizer = wx.BoxSizer(wx.VERTICAL)
+
 		# Translators: The label of the license text which will be shown when NVDA installation program starts.
 		groupLabel = _("License Agreement")
-		sizer = wx.StaticBoxSizer(wx.VERTICAL, self, label=groupLabel)
```

Comment:
note this had to be changed because of https://github.com/wxWidgets/wxWidgets/issues/25058